### PR TITLE
Add id file in resources

### LINKED ISSUE #8: Add resource file
Add a resource file dedicated to the different IDs

### PR DESCRIPTION
**Pull request by Stellucidam**
Adds a file in the resources in order to keep track of all identifiers used in the project.

### CODE REVIEW

**Review by alexandrepiveteau — DISMISSED** [review]
Looks good to me.

Remember to use them as well : writing `@+id/identifier`declares a **new** identifier, and writing `@id/identifier` simply uses an existing declaration (that might be from the resources / another file).

**Comment by alexandrepiveteau** [comment]
@ClarFleu Maybe the syntax `<item name="sample_identifier" type="id"/>` should be used instead.

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
 
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:3.5.3"
+        classpath 'com.android.tools.build:gradle:3.6.2'
```
Comment (alexandrepiveteau) [inline build.gradle:8]: In general, we might want to make upgrades to build files in separate commits and PRs.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Sat Mar 07 17:56:49 CET 2020
+#Sat Apr 04 16:11:20 CEST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```
Comment (alexandrepiveteau) [inline gradle/wrapper/gradle-wrapper.properties:6]: In general, we might want to make upgrades to build files in separate commits and PRs.

**Review by alexandrepiveteau — APPROVED** [review]

**Review by glsubri — APPROVED** [review]
